src/catalog/src/durable/transaction.rs

```diff
@@ -3206,12 +3321,20 @@ mod tests {
         )
         .unwrap();
 
-        table
+        // Ideally, we compare for errors here, but it's hard/impossible to implement PartialEq
+        // for DurableCatalogError.
+        assert!(table
```

Comment:
Might be better to use `assert_ok!` here

Reply:
TIL!